data/baibhav-tiwari.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Baibhav Tiwari",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Universe is cool and so the code",
+  "avatar": "https://avatars.githubusercontent.com/u/75496387?v=4",
```

Comment:
Change the avatar link to this `https://github.com/baibhav-tiwari.png`

Reply:
Yes this is correct and recommended, so that the profile always has the latest GitHub photo  - but can come in the future - so I will merge.  If you wish to update this please raise another Pull Request.

Reply:
@Aadarsh805 sure i'll do it.